libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/grid-areas/dh-market-participant-actor-grid-areas.component.spec.ts

```diff
@@ -0,0 +1,103 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { render, screen } from '@testing-library/angular';
+import userEvent from '@testing-library/user-event';
+import { DhMarketParticipantActorGridAreasComponent } from './dh-market-participant-actor-grid-areas.component';
+import { GridAreaDto } from '@energinet-datahub/dh/shared/domain';
+import { HttpClientModule } from '@angular/common/http';
+import { DhApiModule } from '@energinet-datahub/dh/shared/data-access-api';
+import { getTranslocoTestingModule } from '@energinet-datahub/dh/shared/test-util-i18n';
+
+describe('Grid areas', () => {
```

Comment:
suggestion: use component name

```suggestion
describe(DhMarketParticipantActorGridAreasComponent.name, () => {
```

Reply:
fixed @dzhavat